docs/edge-scripting-api.md

```diff
@@ -0,0 +1,318 @@
+$bunnyClient = new BunnyClient(
+    client: new \Symfony\Component\HttpClient\Psr18Client(),
+);
+
+// Provide the API key available at the "Account Settings > API" section.
+$edgeScriptingApi = new EdgeScriptingAPI(
+    apiKey: '2cebf4f8-4bff-429f-86f6-bce2c2163d7e89fb0a86-a1b2-463c-a142-11eba8811989',
+    client: $bunnyClient,
+);
+```
+
+## Usage
+
+### Code
+
+#### [Get Code](https://docs.bunny.net/reference/getedgescriptcodeendpoint_getcode)
+
+```php
+$edgeScriptingApi->getCode(
+    id: 1,
+);
+```
+
+#### [Set Code](https://docs.bunny.net/reference/uploadedgescriptcodeendpoint_setcode)
+
+```php
+$edgeScriptingApi->setCode(
+    id: 1,
+    body: [
+        'Code' => "import * as BunnySDK from \"https://esm.sh/@bunny.net/edgescript-sdk@0.11.2\";\n\n/**\n * Returns an HTTP response.\n * @param {Request} request - The Fetch API Request object.\n * @return {Response} The HTTP response or string.\n */\nBunnySDK.net.http.serve(async (request: Request): Response | Promise<Response> => {\n  const url = new URL(request.url);\n  return new Response(\"Hello from \" + url.pathname);\n});\n",
+    ],
+);
+```
+
+### Edge Script
+
+#### [List Edge Scripts](https://docs.bunny.net/reference/listedgescriptsendpoint_listedgescriptsbyaccount)
+
+```php
+$edgeScriptingApi->listEdgeScripts(
+    query: [
+        'page' => 1,
+        'perPage' => 1000,
+        'search' => 'Test',
+        'includeLinkedPullzones' => false,
+        'integrationId' => 1234,
+    ],
+);
+```
+
+#### [Add Edge Script](https://docs.bunny.net/reference/createedgescriptendpoint_addscript)
+
+```php
+$edgeScriptingApi->addEdgeScript(
+    body: [
+        'Name' => 'Test Script Modified',
+        'Code' => "import * as BunnySDK from \"https://esm.sh/@bunny.net/edgescript-sdk@0.11.2\";\n\n/**\n * Returns an HTTP response.\n * @param {Request} request - The Fetch API Request object.\n * @return {Response} The HTTP response or string.\n */\nBunnySDK.net.http.serve(async (request: Request): Response | Promise<Response> => {\n  const url = new URL(request.url);\n  return new Response(\"Hello from \" + url.pathname);\n});\n",
+        'ScriptType' => 1,
+        'CreateLinkedPullZone' => true,
+        'LinkedPullZoneName' => 'MyPullZone',
+        'Integration' => [
+            'IntegrationId' => 1234,
+            'RepositorySettings' => [
+                'Id' => 1234,  
+                'Name' => 'Test Repo',  
+                'Private' => true,  
+                'TemplateUrl' => 'https://example.com',  
+            ],
+            'DeployConfiguration' => [
+                'Branch' => 1234,  
+                'InstallCommand' => 'Test Repo',  
+                'BuildCommand' => 'true',  
+                'EntryFile' => 'test.js',  
+                'CreateWorkflow' => false,  
+            ],
+        ],
+    ],
+);
+```
+
+!!! note
+
+    - The key `ScriptType` has the following possible values:
+        - `0` = DNS (DNS scripts)
```

Comment:
The Edge Scripting API can still be used to create DNS scripts (as the previous compute script endpoints were derived from the dashboard).

It however only accepts/uses the `Name` and `ScriptType` keys of the request body.

Add note that using `0` is possible for DNS zones, but it has much less body parameters.